client/components/checkbox/README.md

```diff
@@ -0,0 +1,32 @@
+Checkbox
+========
+
+Checkbox is a React component used to render a checkbox input field. It is essentially an enhanced version of `<input type="checkbox" />`, using Gridicon as check mark.
+
+## Usage
+
+Refer to the following code snippet for a typical usage example:
+
+```jsx
+<Checkbox
+	checked={ this.state.showSubtitles }
+	onChange={ this._toggleShowSubtitles }>
+	Show subtitles
+</Checkbox>
+```
+
+## Props
+
+Props not listed below will be passed automatically to the rendered range input element.
+
+### `checked` (`boolean`)
+
+A boolean indicating whether the checkbox should be checked or not, then the checkmark Gridicon is shown.
```

Comment:
We could skip the first few words here since they are already right above

```js
### `checked` (`boolean`)

Whether or not the checkbox should be checked.
```